pip/pip-330.md

```diff
@@ -0,0 +1,43 @@
+# PIP-330: getMessageById gets all messages
+
+# Motivation
+
+The `org.apache.pulsar.client.admin.Topics` provides `getMessageById(java.lang.String, long, long)` method to get the
+message, which returns one message. If the message id refers to a batch message, we only can get the first message, not
+all messages.
+
+This behavior affects our analysis of messages by the message id.
+
+# Goals
+
+## In Scope
+
+Add a method that returns all messages by message id to the `org.apache.pulsar.client.admin.Topics` interface.
```

Comment:
The server returns the entry, and the client extract the first message and throws the rest?